src/Paket.PowerShell/System.Management.Automation.ps1

```diff
@@ -0,0 +1,2 @@
+# copies System.Management.Automation.dll from the GAC to this folder
+cp ([PSObject].Assembly.Location) .
```

Comment:
Ya, same as FSI - flags *before* the script arg are flags for the script engine, flags *after* the script arg are flags for the script itself.